### PR TITLE
Updated readme to include instructions to install solr using docker

### PR DESCRIPTION
**Pull request by shashigharti**
Fixes #

### Proposed fixes:

Update README to include instructions for installing Solr using docker.

### Features:

- [ ] includes tests covering changes
- [X] includes updated documentation
- [ ] includes user-visible changes
- [ ] includes API changes
- [ ] includes bugfix for possible backport

Please [X] all the boxes above that apply

### CODE REVIEW

**Comment by amercader** [comment]
Thanks @shashigharti , can you replace the 2.0 "Installing Solr manually" instructions with the ones [in master](https://docs.ckan.org/en/latest/maintaining/installing/install-from-source.html#installing-solr-manually) as well? That way they'll be consistent in both versions

**Comment by shashigharti** [comment]
Thanks! @amercader. I have made the changes.

File: doc/maintaining/installing/solr.rst [hunk 1]
```diff
@@ -4,7 +4,79 @@ CKAN uses Solr_ as its search platform, and uses a customized Solr schema file
...
+To check that Solr started you can visit the web interface at http://localhost:8983/solr
+
+.. warning:: The two installation methods above will leave you with a setup that is fine for local development, but Solr should never be exposed publicly in a production site. Pleaser refer to the `Solr documentation <https://solr.apache.org/guide/securing-solr.html>`_ to learn how to secure your Solr instance.
+
+
+If you followed any of the instructions above, the CKAN Solr core will be available at http://localhost:8983/solr/ckan. If for whatever reason you ended up with a different one (eg with a different port, host or core name), you need to change the :ref:`solr_url` setting in your :ref:`config_file` (|ckan.ini|) to point to your Solr server, for example::
+
+       solr_url=http://my-solr-host:8080/solr/ckan-2.9
+
+
```
Comment (amercader) [inline doc/maintaining/installing/solr.rst:77]: Sorry I wasn't clear @shashigharti , no need to keep the Tomcat section, we can remove it as it is superseded by the new "Installing Solr manually" one

Comment (shashigharti) [reply doc/maintaining/installing/solr.rst:77]: Thanks again! I have removed it. Sorry for not understanding quickly.

**Pull request merged**